src/Broadway/Bundle/BroadwayBundle/DependencyInjection/BroadwayExtension.php

```diff
@@ -57,6 +57,7 @@ private function loadCommandBus(array $config, ContainerBuilder $container)
                 'broadway.command_handling.command_bus',
                 'broadway.command_handling.simple_command_bus'
             );
+            $container->removeDefinition('broadway.auditing.command_logger');
```

Comment:
Ah, good question. I forgot to add a comment about it. If no logger is defined in config, the `broadway.auditing.command_logger` is still instantiated because it is tagged as an event listener. Because the logger has not been injected, it's instantiated with an empty string as the first argument, which causes a fatal error.
My solution was to remove the definition, but we could also make the service definition non-public, remove those event listener tags, or only add them if a logger has been provided.